algorithms/ar-bsrh/javascript/binary_Search.js

```diff
@@ -0,0 +1,34 @@
+function binary_Search(items, value) {
```

Comment:
Codacy found an issue: [Function 'binary_Search' has a complexity of 6. Maximum allowed is 5.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6275500)